LayoutTests/http/tests/performance/performance-resource-timing-entries-default-limit.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<script src="../../resources/js-test-pre.js"></script>
```

Comment:
Should use `js-test.js`

Reply:
http/tests/resources doesn't have that variant.

Reply:
We do have it though:
```
<script src="/js-test-resources/js-test.js"></script>
```